src/fn/move-contacts.js

```diff
@@ -21,6 +22,29 @@ module.exports = {
   }
 };
 
+const moveReportsBatch = async (db, contactIds, skip, replacementLineage, contactId, options) => {
+  const queryOptions = {
+    keys: contactIds.map(id => [`contact:${id}`]),
+    include_docs: true,
+    limit: BATCH_SIZE,
+    skip: skip,
```

Comment:
Skip is pretty slow with large data sets :( 
However we can't use `start_key_doc_id` (the faster option) with `keys`, unfortunately (it would be ignored).